exercises/binary/test/test_binary.c

```diff
@@ -0,0 +1,99 @@
+#include "vendor/unity.h"
+#include "../src/binary.h"
+
+void test_b0_is_decimal_0(void)
+{
+   TEST_ASSERT(0 == convert("0"));
```

Comment:
These two tests use `TEST_ASSERT` and the rest use `TEST_ASSERT_EQUAL`. Please standardize on one of them.